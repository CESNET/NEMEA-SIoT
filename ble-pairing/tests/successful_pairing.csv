time TIMESTAMP,macaddr HCI_DEV_MAC,uint8 DATA_DIRECTION,uint8 PACKET_TYPE,bytes PACKET
2019-04-08T13:33:45.570,5c:f3:70:87:46:2b,0,1,052006c52977741d1e
2019-04-08T13:33:45.634,5c:f3:70:87:46:2b,1,4,0e0401052000
2019-04-08T13:33:45.634,5c:f3:70:87:46:2b,0,1,0b200701120012000100
2019-04-08T13:33:45.635,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:33:45.635,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:33:45.636,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:33:45.677,5c:f3:70:87:46:2b,1,4,3e2802010300bc16c7bcbd781c1bff7500420401808e78bdbcc716bcfa042ed7e93901000000000000ab
2019-04-08T13:33:45.685,5c:f3:70:87:46:2b,1,4,3e1a020100011dfb9dd7ab710e02011a0aff4c0010050318bfccbfa9
2019-04-08T13:33:45.686,5c:f3:70:87:46:2b,1,4,3e0c020104011dfb9dd7ab7100aa
2019-04-08T13:33:45.958,5c:f3:70:87:46:2b,1,4,3e2b0201030158a00459b32c1f1eff060001092002ae599549d38b5e62bd489ed1080058d7bbb6041c15b3daa2
2019-04-08T13:33:46.303,5c:f3:70:87:46:2b,1,4,3e2b020100013fd66be7e7d71f0201061bff570100813dea4c97b5254bce967493cc1e3a4f03d7e7e76bd63fb7
2019-04-08T13:33:46.305,5c:f3:70:87:46:2b,1,4,3e1b020104013fd66be7e7d70f0a094d492042616e6420320302e0feb7
2019-04-08T13:33:46.390,5c:f3:70:87:46:2b,1,4,3e2b02010000af14da1ab1cc1f02011e1bff7500420401404eccb11ada14afceb11a6014ae01000000000000a3
2019-04-08T13:33:46.492,5c:f3:70:87:46:2b,1,4,3e2b02010400af14da1ab1cc1f020a091b085b54565d2053616d73756e672035205365726965732028333229a4
2019-04-08T13:33:48.364,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:33:48.367,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:33:48.367,5c:f3:70:87:46:2b,0,1,0b200700600030000001
2019-04-08T13:33:48.369,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:33:48.369,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:33:48.370,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:33:50.161,5c:f3:70:87:46:2b,1,4,3e2b020100013fd66be7e7d71f0201061bff570100813dea4c97b5254bce967493cc1e3a4f03d7e7e76bd63fb9
2019-04-08T13:33:50.161,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:33:50.164,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:33:50.164,5c:f3:70:87:46:2b,0,1,0d20196000600000013fd66be7e7d7001800280000002a0000000000
2019-04-08T13:33:50.166,5c:f3:70:87:46:2b,1,4,0f0400010d20
2019-04-08T13:33:51.451,5c:f3:70:87:46:2b,1,4,3e130100400000013fd66be7e7d7270000002a0005
2019-04-08T13:33:51.451,5c:f3:70:87:46:2b,0,1,1620024000
2019-04-08T13:33:51.454,5c:f3:70:87:46:2b,1,4,0f0400011620
2019-04-08T13:33:51.479,5c:f3:70:87:46:2b,1,2,402010000c00050012a40800180028000000f401
2019-04-08T13:33:51.529,5c:f3:70:87:46:2b,1,4,3e0c040040000100000000000000
2019-04-08T13:33:51.529,5c:f3:70:87:46:2b,0,2,40000b00070006000104002d100d0f
2019-04-08T13:33:51.529,5c:f3:70:87:46:2b,0,2,40000a000600050013a402000000
2019-04-08T13:33:51.529,5c:f3:70:87:46:2b,0,1,13200e4000180028000000f40100000000
2019-04-08T13:33:51.532,5c:f3:70:87:46:2b,1,4,0f0400011320
2019-04-08T13:33:51.621,5c:f3:70:87:46:2b,0,2,4000070003000400020502
2019-04-08T13:33:51.627,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:51.674,5c:f3:70:87:46:2b,1,2,4020070003000400038000
2019-04-08T13:33:51.674,5c:f3:70:87:46:2b,0,2,40000b0007000400100100ffff0028
2019-04-08T13:33:51.675,5c:f3:70:87:46:2b,1,2,40200b000700060002030001100505
2019-04-08T13:33:51.675,5c:f3:70:87:46:2b,0,2,400015001100060003d657fbbcd693535eb0a4afff8aec0cfa
2019-04-08T13:33:51.724,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:51.772,5c:f3:70:87:46:2b,1,2,4020180014000400110601000700001808000b0001180c0016000a18
2019-04-08T13:33:51.772,5c:f3:70:87:46:2b,0,2,40000b0007000400101700ffff0028
2019-04-08T13:33:51.772,5c:f3:70:87:46:2b,1,2,402015001100060003e49e56cd680f084792852ffaa76777ab
2019-04-08T13:33:51.772,5c:f3:70:87:46:2b,0,2,400015001100060004536eb41a878bb1457e6a801a59034b39
2019-04-08T13:33:51.821,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:51.918,5c:f3:70:87:46:2b,1,2,40201a0016000400111417001c00000710af090018211235000030150000
2019-04-08T13:33:51.918,5c:f3:70:87:46:2b,0,2,40000b0007000400101d00ffff0028
2019-04-08T13:33:51.919,5c:f3:70:87:46:2b,1,2,40201500110006000427e4001bb71e0f223e1539758207bb79
2019-04-08T13:33:51.919,5c:f3:70:87:46:2b,0,1,19201c4000000000000000000000000467daf818fc7c08791a89c05c611932
2019-04-08T13:33:51.920,5c:f3:70:87:46:2b,1,4,3e0a0300400027000000f401
2019-04-08T13:33:51.921,5c:f3:70:87:46:2b,1,4,0f0400011920
2019-04-08T13:33:52.017,5c:f3:70:87:46:2b,1,4,13050140000100
2019-04-08T13:33:52.211,5c:f3:70:87:46:2b,1,4,080400400001
2019-04-08T13:33:52.211,5c:f3:70:87:46:2b,1,2,40201500110006000681405c7f2f56ba85bdccf8eb2ba216b1
2019-04-08T13:33:52.212,5c:f3:70:87:46:2b,1,2,40200f000b00060007ad0dc5344093b38aded9
2019-04-08T13:33:52.212,5c:f3:70:87:46:2b,1,2,40201500110006000a79d814d7a76ef25db564b043a3ba4e89
2019-04-08T13:33:52.213,5c:f3:70:87:46:2b,0,2,400015001100060006f2dfad997c70938297fff3c463aa622d
2019-04-08T13:33:52.213,5c:f3:70:87:46:2b,0,2,40000f000b0006000741caa12573472e1cac94
2019-04-08T13:33:52.213,5c:f3:70:87:46:2b,0,2,40001500110006000a38019661dd67495c8d85c1c81ef17e0a
2019-04-08T13:33:52.260,5c:f3:70:87:46:2b,1,2,40201b002000040011061d002300111824002600021827002c000d182d0051
2019-04-08T13:33:52.260,5c:f3:70:87:46:2b,1,2,4010090000e0fe52006600e1fe
2019-04-08T13:33:52.261,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:52.262,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:52.348,5c:f3:70:87:46:2b,0,2,40000b0007000400106700ffff0028
2019-04-08T13:33:52.406,5c:f3:70:87:46:2b,1,2,40200900050004000110ffff0a
2019-04-08T13:33:52.447,5c:f3:70:87:46:2b,0,2,4000090005000400120b000200
2019-04-08T13:33:52.455,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:52.503,5c:f3:70:87:46:2b,1,2,402005000100040013
2019-04-08T13:33:52.503,5c:f3:70:87:46:2b,0,2,40000700030004000a0300
2019-04-08T13:33:52.601,5c:f3:70:87:46:2b,1,2,40200e000a0004000b4d492042616e642032
2019-04-08T13:33:52.601,5c:f3:70:87:46:2b,0,2,40000700030004000a0500
2019-04-08T13:33:52.650,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:33:52.698,5c:f3:70:87:46:2b,1,2,40200700030004000b0000
2019-04-08T13:33:52.699,5c:f3:70:87:46:2b,0,2,40000700030004000a1600
2019-04-08T13:33:52.796,5c:f3:70:87:46:2b,1,2,40200c00080004000b01570104000001
