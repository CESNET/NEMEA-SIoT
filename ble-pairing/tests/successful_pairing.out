2019-04-08T13:33:52.211,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,0,0,1,0
